src/components/app.tsx

```diff
@@ -649,9 +663,12 @@ export class App extends React.Component<AppProps, AppState> {
 
     launchSensorConnector = () => {
         this.setState({ statusMessage: "Launching SensorConnector...", notRespondingModal: false });
-        this.state.sensorManager!.requestSleep();
-        // pause before attempting to reload SensorConnector
-        setTimeout(() => this.state.sensorManager!.requestWake(), SLEEP_WAKE_DELAY_SEC * 1000);
+        const { sensorManager } = this.state;
+        if (sensorManager) {
+            sensorManager.requestSleep();
+            // pause before attempting to reload SensorConnector
+            setTimeout(() => sensorManager.requestWake(), SLEEP_WAKE_DELAY_SEC * 1000);
```

Comment:
Seems like we might need an additional check inside the `setTimeout()` to account for the case in which the SensorManager has been deleted/changed in the interim. Occurs here and elsewhere in this file.

Reply:
Yup.  Added protection around it.